htdocs/knowledgemanagement/class/knowledgerecord.class.php

```diff
@@ -220,7 +220,7 @@ public function __construct(DoliDB $db)
 	 */
 	public function create(User $user, $notrigger = 0)
 	{
-		return $this->createCommon($user, $notrigger);
+		return $this->createCommon($user, $notrigger, 'KNOWLEDGERECORD_CREATE');
```

Comment:
According to code in createCommon, we already have

$result = $this->call_trigger(strtoupper(get_class($this)).'_CREATE', $user);

So trigger with name KNOWLEDGERECORD_CREATE should already be called with no need to provide the name 'KNOWLEDGERECORD_CREATE' as parameter. did i miss something ?

Reply:
No sorry it's me

Reply:
it's done